File: 1-Easy/Onboarding/onboarding.c
#include <stdio.h>

int main() {
    char enemy1[257], enemy2[257];
    int dist1, dist2;

    // game loop
    while (1) {
        scanf("%s%d%s%d", enemy1, &dist1, enemy2, &dist2);
        printf("%s\n", (dist1 < dist2) ? enemy1 : enemy2);
    }

    return 0;
}

File: 1-Easy/Onboarding/onboarding.cs
using System;

class Player
{
    static void Main(string[] args)
    {
        // game loop
        while (true)
        {
            string enemy1 = Console.ReadLine();
            int dist1 = int.Parse(Console.ReadLine());
            string enemy2 = Console.ReadLine();
            int dist2 = int.Parse(Console.ReadLine());

            Console.WriteLine((dist1 < dist2) ? enemy1 : enemy2);
        }
    }
}